heroku.md

```diff
@@ -26,7 +26,17 @@ Here's how to deploy your implementation of, say, [C$50 Finance](https://cs50.ha
     git remote add origin https://github.com/username/finance.git
     ```
 
-1. In the `requirements.txt` file inside of your `finance` directory, add `gunicorn`, `psycopg2` and `requests` each on separate lines.
+1. In the `requirements.txt` file inside of your `finance` directory, add `gunicorn`, `psycopg2`, `requests` and `SQLAlchemy==1.3.23`, each on separate lines. Your file should then resemble:
+
+```
+cs50
+Flask
+Flask-Session
+gunicorn
+psycopg2
+requests
+SQLAlchemy==1.3.23
```

Comment:
Why the `SQLAlchemy==1.3.23` fixation? Since we're not fixating others, should probably be `>=`?

Reply:
So the issue is that Heroku is still using an unsupported postgres dialect, and SQLAlchemy 1.3.23 was the last version that supported that.
[
https://help.heroku.com/ZKNTJQSK/why-is-sqlalchemy-1-4-x-not-connecting-to-heroku-postgres](
https://help.heroku.com/ZKNTJQSK/why-is-sqlalchemy-1-4-x-not-connecting-to-heroku-postgres)